packages/js/core/src/algorithms/get-implementations.ts

```diff
@@ -23,7 +23,7 @@ export const getImplementations = Tracer.traceFunc(
     const addAllImplementationsFromImplementationsArray = (
       implementationsArray: readonly InterfaceImplementations<Uri>[],
       wrapperInterfaceUri: Uri
-    ) => {
+    ): Result<true, Error> => {
```

Comment:
Since this is always true when it's `ok`, maybe it should be Result<undefined, Error>
Returning would look like: ResultOk(undefined)
Then users could know they can always ignore result.data, though I guess it's pretty much the same if the type is true.
Just a minor nitpick

Reply:
That's a good point. You're 100% right. I changed it.